packages/backend/package.json

```diff
@@ -51,6 +51,16 @@
           "maximum": 65535,
           "description": "Port on which the API is listening (requires restart of extension)"
         },
+        "ai-lab.inferenceRuntime": {
+          "type": "string",
+          "default": "llama-cpp",
+          "enum": [
+            "llama-cpp",
```

Comment:
Is there a way we can have this dynamically generated ?

Reply:
no there is not